.github/workflows/ci.yml

```
@@ -75,7 +76,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip setuptools wheel
-        python -m pip install --upgrade ".[test]"
+        python -m pip install --upgrade --verbose -Ccmake.verbose=1 -Clogging.level=INFO ".[test]"
```

```suggestion
        python -m pip install --upgrade ".[test]"
        python -m pip uninstall wasserstein
        python -m pip install --upgrade --verbose -Ccmake.verbose=1 -Clogging.level=INFO .
```

So while I don't know if this makes any sense, `pytest` isn't getting installed when it previously(?) was, so we could first install things as before, then uninstall `wasserstein` and then try to install _only_ it with the more informative logging.